code/modules/mob/living/simple_animal/abnormality/he/woodsman.dm

```diff
@@ -0,0 +1,300 @@
+	layer = BELOW_OBJ_LAYER
+	maxHealth = 1433
+	health = 1433
+	ranged = TRUE
+	attack_verb_continuous = "chops"
+	attack_verb_simple = "chop"
+	faction = list("hostile")
+	attack_sound = 'sound/abnormalities/woodsman/woodsman_attack.ogg'
+	stat_attack = HARD_CRIT
+	melee_damage_lower = 15
+	melee_damage_upper = 30
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.5, WHITE_DAMAGE = 0.8, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 1)
+	vision_range = 14
+	aggro_vision_range = 20
+	attack_action_types = list(/datum/action/innate/abnormality_attack/woodsman_flurry)
+	can_buckle = TRUE
+	can_breach = TRUE
+	threat_level = HE_LEVEL
+	start_qliphoth = 1
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = 45,
+						ABNORMALITY_WORK_INSIGHT = 45,
+						ABNORMALITY_WORK_ATTACHMENT = list(50, 60, 70, 80, 90),
+						ABNORMALITY_WORK_REPRESSION = 45
+						)
+	work_damage_amount = 10
+	work_damage_type = WHITE_DAMAGE
+
+	//base_pixel_x = -16
+	pixel_x = -16
+
+
+	ego_list = list(
+		/datum/ego_datum/weapon/logging,
+		/datum/ego_datum/armor/logging
+		)
+
+	var/meltdown_breach = FALSE
+	var/flurry_cooldown = 0
+	var/flurry_cooldown_time = 15 SECONDS
+	var/flurry_count = 7
+	var/flurry_small = 12
+	var/flurry_big = 60 // It was requested that he beats their ass harder
+	var/can_act = TRUE
+
+/datum/action/innate/abnormality_attack/woodsman_flurry
+	name = "Deforestation"
+	icon_icon = 'ModularTegustation/Teguicons/tegumobs.dmi'
+	button_icon_state = "training_rabbit"
+	chosen_message = "<span class='colossus'>You will now attempt to fell all hearts in your path.</span>"
+	chosen_attack_num = 1
+
+/mob/living/simple_animal/hostile/abnormality/woodsman/Move()
+	if(!can_act)
+		return FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/woodsman/AttackingTarget(atom/attacked_target)
+	if(!can_act)
+		return FALSE
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		if(H.stat == DEAD || (H.health <= HEALTH_THRESHOLD_DEAD && HAS_TRAIT(H, TRAIT_NODEATH)))
+			Heal(H)
+			return ..()
+	if(isliving(target) && flurry_cooldown <= world.time && get_dist(src, target) <= 2 && prob(30))
+		Woodsman_Flurry(target)
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/woodsman/PickTarget(list/Targets) // We attack corpses first if there are any
+	var/list/highest_priority = list()
+	var/list/lower_priority = list()
+	for(var/mob/living/L in Targets)
+		if(!CanAttack(L))
+			continue
+		if((L.health < 0 || L.stat == DEAD) && ishuman(L))
+			highest_priority += L
+		else
+			lower_priority += L
+	if(LAZYLEN(highest_priority))
+		return pick(highest_priority)
+	if(LAZYLEN(lower_priority))
+		return pick(lower_priority)
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/woodsman/proc/Heal(mob/living/carbon/human/body)
+	src.adjustBruteLoss(-666) // Actually just the conversion of health he heals scaled to equivalent health that Helper has.
+	for(var/obj/item/organ/O in body.getorganszone(BODY_ZONE_CHEST, TRUE))
```

Comment:
This is pointless since you already gib the person.

Reply:
This rips their heart out and deletes it, as the woodsman "takes" their heart.